### ISSUE TITLE
What is the status of this project?

### ISSUE THREAD

**Issue by JeffLuckett** [Netflix/fast_jsonapi#452]
I've had my eye on this project since it was announced back in '18.  I work on a large JSON API that uses ActiveModelSerializers, and we've been "stuck" on the 0.8 version of the gem ever since that project killed the 0.9 version.  We know we've got work to do to get current, and I have long considered swapping it out for fast_jsonapi, since it's so much faster and we've got work to do anyway.  

I've just recently begun a side project, and had hoped to use this library for my serialization.  It's a good fit for that project, and would give me the familiarity with the Gem to decide whether I want to bring it into my employer's API as the AMS replacement.  However, I've noticed the lack of activity here ... no PR's merged since last March, and that gives me pause.

Now I've also noticed that some community members have created a fork (https://github.com/fast-jsonapi/fast_jsonapi) and are actively working on it since October given the sudden radio silence here.  However, since they can't publish releases to RubyGems, you are required to install via Github.  I don't mind that for my side project, but would be reluctant to do such for my employer's project.

Is it the intention of Netflix and the maintainers to allow this project to die on the vine?  Might you possibly instead hand over the reins of the "official" repo to these new maintainers so that new releases can be posted to RubyGems, and developers can have the confidence to include this promising library in their projects?

Please reach out to the folks at https://github.com/fast-jsonapi/fast_jsonapi and figure out the best path forward.  If Netflix doesn't currently have the resources to maintain this gem, perhaps you could give the maintainers of the fork Collaborator status here and allow them to merge their fork back in and continue to develop the "official" library?

If it's not possible to give them commit rights in this Repo, how about just shutting this project down, and redirecting people to the fork, allow them to "unfork" and become the canonical repo.  This is a great project for those of us who are trying to develop performant JSON API's, please don't let it die.

**Comment by chrisgeek** [comment]
I had precisely the same question, I have kept an eye on this repo for a while now, I'm not sure what to make of the continued silence of the maintainers, anyways, I moved on to [jsonapi-rb](http://jsonapi-rb.org/guides/getting_started/rails.html)

**Comment by mhluska** [comment]
@chrisgeek you've moved on to an alternative with last commit 3 years ago?

**Comment by chrisgeek** [comment]
Actually I had to settle for a working version of AMS for now (development), jsonapi-rb seems abandoned too.

**Comment by stas** [comment]
We're here: https://github.com/fast-jsonapi/fast_jsonapi :raising_hand_man:

**Comment by cesarjr** [comment]
Oh gosh! I was about to test this one to migrate from ams 0.9.x to it. I think I'll give up of rest and migrate to graphql.

**Comment by stas** [comment]
@cesarjr last thing this project needs is folks sharing this sort of attitude. Please, next time spare us all from your opinions on not being happy that people can't put more of their free time into software that you don't have to pay for. :disappointed:

**Comment by kapilnarula** [comment]
Yes this project seems abandoned.

However, the community has created a fork which is being maintained at
 https://github.com/fast-jsonapi/fast_jsonapi

**Comment by cesarjr** [comment]
Hi @stas ! I'm really sorry about that. It wasn't my intent and I understand your point of view.

**Comment by StanBright** [comment]
Yes, it will be great if there was an official message that this gem is not maintained anymore and a link to https://github.com/fast-jsonapi/fast_jsonapi on the top of the README.

**Comment by stas** [comment]
We are preparing a rename for the fork and we could use your help picking up the name.

Please leave your thoughts at: https://github.com/fast-jsonapi/fast_jsonapi/issues/86

Thank you!
